stdlib/date.rb

```diff
@@ -547,6 +547,10 @@ def next_month(n = 1)
     }
   end
 
+  def next_year(years=1)
```

Comment:
Add spaces around `=` please

Reply:
There aren't consistent conventions for [spaces](https://github.com/opal/opal/blob/0374448b034e88662b329e95af0aa77cba9d40d3/stdlib/observer.rb#L126) [around](https://github.com/opal/opal/blob/0374448b034e88662b329e95af0aa77cba9d40d3/stdlib/observer.rb#L167) [default](https://github.com/opal/opal/blob/0374448b034e88662b329e95af0aa77cba9d40d3/stdlib/pp.rb#L22) args. Those 3 are just what I noticed in `stdlib`. There are probably more.

It's reasonable to ask contributors to use existing conventions, but it'd be courteous to them to fix the conventions used in your project before asking those contributors to do more work to conform to them. When they're not written down anywhere (whether it's a line in the [contribution guidelines](https://github.com/opal/opal/blob/master/CONTRIBUTING.md#submitting-a-pull-request) or a linter either in CI or a GitHub PR bot), the only way people know what your conventions are is by looking at existing code. If existing conventions are not what you're looking for, it's not the responsibility of the contributor to fix it.